src/components/ResultsComponent.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { IonImg, IonSpinner, IonButton, IonGrid, IonRow, IonCol, IonAlert } from '@ionic/react';
+import CSS from 'csstype';
+import SingleResult from './SingleResultComponent';
+
+interface Props {}
+
+interface State {
+    showAlert: boolean
+}
+
+export default class ResultsComponent extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+    }
+
+    state = {
+        showAlert: false
+    }
+
+    private stockPhoto = 'https://images.unsplash.com/photo-1597223557154-721c1cecc4b0?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80'
+
+    public count = [1,2,3,4]
+
+    public triggerAlert = () => {
+        this.setState(current => ({showAlert: !current.showAlert}))
+        
+    }
+
+    render() {
+        return (
+            <div style={this.ResultComponentStyle}>
+            <IonGrid>
+                <IonRow>
+                    <h1>We found a high probability match</h1>
+                </IonRow>
+                {this.count.map(num => <SingleResult key={num} photo={this.stockPhoto} />)}
+                <IonRow className="ion-justify-content-center">
+                    <div style={this.ResultButtonStyle}>
+                        <IonButton color="danger" onClick={this.triggerAlert}>Contact Law Enforcement</IonButton>
+                        <IonAlert 
+                            isOpen={this.state.showAlert} 
+                            onDidDismiss={() => this.triggerAlert}
+                            header={'Call Law Enforcement'}
+                            subHeader={'###-###-####'}
+                            buttons={['Call', 'Cancel']}
+                            message={'Please refrain from intervening in suspicious activity and contact local officials'}
+                        />
+                    </div>
+                </IonRow>
+            </IonGrid>
+            </div>
+        );
+      };
+
+    private ResultComponentStyle: CSS.Properties = {
+        color: 'black',
+        textAlign: 'center',
+        width: '100%',
+        height: '100%',
+        // padding: '1rem 0 rem 0',
+    }
+
+    private ImageContainerStyle: CSS.Properties = {
+        textAlign: 'center',
+        padding: '2rem'
+    }
+
+    private SpinnerStyle: CSS.Properties = {
+        transform: 'scale(1.5)'
+    }
+
+    private ResultButtonStyle: CSS.Properties = {
```

Comment:
Being class property names, these should be camel case. Any variables, properties, arguments, etc. should be camel case in JavaScript. Class names are the exception where they're pascal case (camel case but first letter is also capitalized).
```suggestion
    private resultComponentStyle: CSS.Properties = {
        color: 'black',
        textAlign: 'center',
        width: '100%',
        height: '100%',
        // padding: '1rem 0 rem 0',
    }

    private imageContainerStyle: CSS.Properties = {
        textAlign: 'center',
        padding: '2rem'
    }

    private spinnerStyle: CSS.Properties = {
        transform: 'scale(1.5)'
    }

    private resultButtonStyle: CSS.Properties = {
```

Reply:
I'll make those changes rn!